File: spec/code_samples/Node request/orders@{orderId}/parameters.js
var request = require("request");

var options = {
  method: 'PARAMETERS',
  url: 'http://ec2-3-222-208-106.compute-1.amazonaws.com:8086/orders/%7BorderId%7D'
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: spec/code_samples/Python python3/orders@{orderId}/parameters.py
import http.client

conn = http.client.HTTPConnection("ec2-3-222-208-106.compute-1.amazonaws.com:8086")

conn.request("PARAMETERS", "/orders/%7BorderId%7D")

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))